qiskit_optimization/algorithms/cobyla_optimizer.py

```diff
@@ -117,9 +118,14 @@ def solve(self, problem: QuadraticProgram) -> OptimizationResult:
         """
         self._verify_compatibility(problem)
 
+        # we deal with minimization in the optimizer, so turn the problem to minimization
+        max2min = MaximizeToMinimize()
+        original_problem = problem
+        problem = self._convert(problem, max2min)
+
         # construct quadratic objective function
         def objective(x):
-            return problem.objective.sense.value * problem.objective.evaluate(x)
+            return problem.objective.evaluate(x)
```

Comment:
This could be a lambda, since it is a one liner. But this is a matter of taste, so you can keep it as is.

Reply:
We don't need to make a function. It is sufficient to just pass `problem.objective.evaluate` to `fmin_cobyla` as `x = fmin_cobyla(problem.objective.evaluate, ...)`

Reply:
Oh yeah, that's rights! Shame on me, there no need even for lambda!

Reply:
It is just passed problem.objective.evaluate now.